tests/urlpatterns_reverse/tests.py

```diff
@@ -843,10 +890,66 @@ def test_exceptions(self):
 
 
 class IncludeTests(SimpleTestCase):
+    def setUp(self):
```

Comment:
is there a need to instantiate these for each test? could they be class attributes instead?